src/lib/pageview.js

```diff
@@ -10,13 +10,42 @@
  *   * It runs _before_ our core bundle (for supported browsers).
  */
 
+import {getCLS, getFID, getLCP} from 'web-vitals';
 import {dimensions, id, version} from 'webdev_analytics';
 import {localStorage} from './utils/storage';
 
+/**
+ * See: https://github.com/GoogleChrome/web-vitals#using-analyticsjs
+ * @param {Object} metric
+ */
+function sendToGoogleAnalytics({name, delta, id}) {
+  ga('send', 'event', {
+    eventCategory: 'Web Vitals',
+    eventAction: name,
+    // Google Analytics metrics must be integers, so the value is rounded.
+    // For CLS the value is first multiplied by 1000 for greater precision
+    // (note: increase the multiplier for greater precision if needed).
+    eventValue: Math.round(name === 'CLS' ? delta * 1000 : delta),
+    // The `id` value will be unique to the current page load. When sending
+    // multiple values from the same page (e.g. for CLS), Google Analytics can
+    // compute a total by grouping on this ID (note: requires `eventLabel` to
+    // be a dimension in your report).
+    eventLabel: id,
+    // Use a non-interaction event to avoid affecting bounce rate.
+    nonInteraction: true,
+  });
+}
+
 // eslint-disable-next-line
 window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
 ga('create', id);
 ga('set', 'transport', 'beacon');
+ga('set', 'page', window.location.pathname);
```

Comment:
Note, I added this change for consistency with the logic [here](https://github.com/GoogleChrome/web.dev/blob/fb3d450613ecd5ec68f84b3407aaeeca4675746d/src/lib/loader.js#L170) (which will remove query params from the URLs of pages in your GA reports).

I can remove this though if you want to keep this PR just to the scope of adding `web-vitals`.